.github/workflows/dockerhub.yml

```diff
@@ -29,16 +29,15 @@ jobs:
     # Build the Docker image
     - name: Build
       run: |        
-        docker build -t "$DOCKER_ORGANIZATION"/"$IMAGE":"$GITHUB_SHA" \
+        docker build -t "$DOCKER_ORGANIZATION"/"$IMAGE":$(echo ${GITHUB_SHA} | cut -c1-8)\
           --build-arg GITHUB_SHA="$GITHUB_SHA" \
           --build-arg GITHUB_REF="$GITHUB_REF" .
```

Comment:
does the variable $GITHUB_REF actually contain anything? I only see $GITHUB_SHA defined in `env`

Reply:
I left this from the default Workflow provided by Google for GKE deployment. That being said, we are not using either of these variables so I might as well remove them.